### ISSUE TITLE
[bug]: autoconfig rules not displaying in summary message

### ISSUE THREAD

**Issue by chrisguida** [Start9Labs/embassy-os#1371]
<img width="709" alt="Screen Shot 2022-03-31 at 12 08 55 PM" src="https://user-images.githubusercontent.com/7445670/161121832-926da8b3-21c6-4b04-9453-123b552da304.png">
<img width="720" alt="Screen Shot 2022-03-31 at 12 09 01 PM" src="https://user-images.githubusercontent.com/7445670/161121838-c853d89e-fe09-4d3c-9ff5-f960a6fb7e42.png">

with this rules.yaml file:
```
- rule: '''users.*.name = "electrs"'
  description: 'Must have an RPC user named "electrs"'
  suggestions:
    - PUSH:
        to: "users"
        value:
          name: electrs
          allowed-calls: []
          fetch-blocks: false
    - SET:
        var: 'users.[first(item => ''item.name = "electrs")].password'
        to-entropy:
          charset: "a-z,A-Z,0-9"
          len: 22
- rule: '!(users.[first(item => ''item.name = "electrs")].fetch-blocks?)'
  description: 'RPC user "electrs" must have "Fetch Blocks" disabled'
  suggestions:
    - SET:
        var: 'users.[first(item => ''item.name = "electrs")].fetch-blocks'
        to-value: false
- rule: '''users.[first(item => ''item.name = "electrs")].allowed-calls.* = "estimatesmartfee"'
  description: 'RPC user "electrs" must have "estimatesmartfee" enabled'
  suggestions:
    - PUSH:
        to: 'users.[first(item => ''item.name = "electrs")].allowed-calls'
        value: "estimatesmartfee"
- rule: '''users.[first(item => ''item.name = "electrs")].allowed-calls.* = "getblockchaininfo"'
  description: 'RPC user "electrs" must have "getblockchaininfo" enabled'
  suggestions:
    - PUSH:
        to: 'users.[first(item => ''item.name = "electrs")].allowed-calls'
        value: "getblockchaininfo"
- rule: '''users.[first(item => ''item.name = "electrs")].allowed-calls.* = "getmempoolentry"'
  description: 'RPC user "electrs" must have "getmempoolentry" enabled'
  suggestions:
    - PUSH:
        to: 'users.[first(item => ''item.name = "electrs")].allowed-calls'
        value: "getmempoolentry"
- rule: '''users.[first(item => ''item.name = "electrs")].allowed-calls.* = "getnetworkinfo"'
  description: 'RPC user "electrs" must have "getnetworkinfo" enabled'
  suggestions:
    - PUSH:
        to: 'users.[first(item => ''item.name = "electrs")].allowed-calls'
        value: "getnetworkinfo"
- rule: '''users.[first(item => ''item.name = "electrs")].allowed-calls.* = "getrawmempool"'
  description: 'RPC user "electrs" must have "getrawmempool" enabled'
  suggestions:
    - PUSH:
        to: 'users.[first(item => ''item.name = "electrs")].allowed-calls'
        value: "getrawmempool"
- rule: '''users.[first(item => ''item.name = "electrs")].allowed-calls.* = "getrawtransaction"'
  description: 'RPC user "electrs" must have "getrawtransaction" enabled'
  suggestions:
    - PUSH:
        to: 'users.[first(item => ''item.name = "electrs")].allowed-calls'
        value: "getrawtransaction"
```

All rpc calls should be displayed at the top, alongside "users: Added new entry"

**Comment by dr-bonez** [comment]
I'm not sure the information necessary is available in the api. Will need investigate the best way to handle this.

**Comment by MattDHill** [comment]
I'm of the opinion that the summary is sufficient, and if a user cares to investigate the specifics, they are free to follow the "edited" flags and check it out. For example, say we added a new "user" to some service. Displaying everything about the user in the summary seems unnecessary and almost certainly unwanted. The summary is that we added a new user, the details of that user can be found in the user object below.
